### ISSUE TITLE
Import naming issue (supersuit=3.3.3 pettingzoo==1.15.0)

### ISSUE THREAD

**Issue by hvpeteet** [Farama-Foundation/SuperSuit#132]
TLDR: cannot import name 'aec_to_parallel' (supersuit=3.3.3 pettingzoo==1.15.0) due to renamed functions in pettingzoo

Repro: 
```bash
pip install pettingzoo==1.14.0
pip install supersuit==3.3.3
```
```python
from supersuit.utils.wrapper_chooser import WrapperChooser
```
```diff
-ImportError: cannot import name 'aec_to_parallel' from 'pettingzoo.utils.conversions'
```

### The bug:

1. pettingzoo 1.15.0 changed the name of 2 functions `to_parallel` and `from_parallel` to `aec_to_parallel` and `aec_from_parallel` https://github.com/Farama-Foundation/PettingZoo/releases/tag/1.15.0
2. supersuit 3.3.3 picked up this change and changed their imports https://github.com/Farama-Foundation/SuperSuit/blob/2f31857463843467a98783ced83194b66dbd7059/supersuit/utils/wrapper_chooser.py#L3
3. supersuit 3.3.3 didn't update their min version of pettingzoo https://github.com/Farama-Foundation/SuperSuit/blob/2f31857463843467a98783ced83194b66dbd7059/setup.py#L30

Consequence: when running pettingzoo < 1.15.0 and supersuit > 3.3.3 you can get an import error when using WrapperChooser or anything that depends on it (action_lambda, etc...)

### Possible fixes (not sure what y'all would prefer)
* SuperSuit 3.3.3 should depend on pettingzoo>=1.15.0
* OR SuperSuit 3.3.3 should do conditional imports for the new name and if that fails fall back to the old names and alias them. E.g. something like 
    ```
    try:
        from pettingzoo.utils.conversions import aec_to_parallel, parallel_to_aec
    except ImportError:
        from pettingzoo.utils.conversions import to_parallel as aec_to_parallel
        from pettingzoo.utils.conversions import from_parallel as parallel_to_aec
    ```
* OR reopen this issue under pettingzoo and make sure they provide backwards compatible aliases for the old function names

**Comment by benblack769** [comment]
I'm sorry to hear you ran into this issue. I think we would prefer to just fix the minimum version of pettingzoo in supersuit. Pettingzoo and supersuit are very closely tied libraries and often have concurrent release dates, so it makes sense for supersuit releases to have pretty strict dependencies on pettingzoo versions.

In the meantime, I would try to use the latest versions of both packages whenever possible. If there is some issue preventing you from using the latest versions of both packages, please let us know.

**Comment by jkterry1** [comment]
Hey, I'm closing this due to inactivity, please let us know if you have any questions
